easybuild/easyconfigs/h/HMMER/HMMER-3.1b2-foss-2016a.eb

```diff
@@ -28,16 +28,16 @@ description = """HMMER is used for searching sequence databases for homologs of
 toolchain = {'name': 'foss', 'version': '2016a'}
 
 source_urls = ['http://eddylab.org/software/hmmer%(version_major)s/%(version)s/']
-sources = ['hmmer-%(version)s-linux-intel-x86_64.tar.gz']
+sources = ['hmmer-%(version)s.tar.gz']
+
+runtest = 'check'
 
```

Comment:
Well, strictly speaking, order doesn't matter, but we tend to specify easyconfig parameters according to the steps they're used in'

`runtest` specifies how the included tests should be run (here with `make check`), which is done between the `build` and `install` steps, so I'm specifying it before `installopts` following the order of steps.